recipes/run-dbcan/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "run-dbcan" %}
+{% set version = "2.0.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/ae/87/431b9b651544ead5b0889c39e6fddb9e1dd98e773c2ed5c229e18e063ee6/run_dbcan-2.0.11.tar.gz"
```

Comment:
If you use this URL https://pypi.io/packages/source/r/run_dbcan/run_dbcan-2.0.11.tar.gz and with the version variable {{version}} than the bot can update it easily

Reply:
The link was down yesterday. It works now!